jdi-light-angular-tests/src/main/java/io/github/com/pages/sections/MenuSection.java

```diff
@@ -0,0 +1,26 @@
+package io.github.com.pages.sections;
+
+import com.epam.jdi.light.angular.elements.common.NestedDropdownMenu;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.UI;
+import com.epam.jdi.light.ui.html.elements.common.Text;
+
+public class MenuSection {
+
+    @UI("#basic-menu-button")
+    public static NestedDropdownMenu basicMenu;
+
+    @UI("#menu-with-icons-button")
+    public static NestedDropdownMenu menuWithIcons;
+
+    @UI("#nested-menu-button")
+    public static NestedDropdownMenu nestedMenu;
+
+    @UI("#basic-menu-selected-option")
```

Comment:
Just as a hint. If the class name matches the id, then we can remove the @UI("#id") annotation. For example,
basicMenuSelectedOption corresponds to #basic-menu-selected-option.

Reply:
Got you, thanks.